format.diff

```diff
@@ -0,0 +1,18 @@
+diff --git a/symengine/pow.cpp b/symengine/pow.cpp
```

Comment:
Please remove this file and the binary file `symengine/tests/basic/test_cse` and then squash the commits so that they don't appear in the history

Reply:
Okay.. Will do so..